fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/common/RecordSerializer.java

```diff
@@ -115,14 +116,28 @@ M deserialize(@Nonnull RecordMetaData metaData, @Nonnull Tuple primaryKey,
         DECRYPT_SERIALIZED_RECORD("decrypt serialized record");
 
         private final String title;
-        Events(String title) {
+        private final String logKey;
+
+        Events(String title, String logKey) {
             this.title = title;
+            this.logKey = ((logKey != null) ? logKey : this.name()).toLowerCase(Locale.ROOT);
```

Comment:
It seems a little weird to me that the log key is lower-cased even if the user explicitly sets a non-`null` one (or, other words, there's no way to force a capital letter in). Maybe that's what we want, but it's kind of a strange contract. (Lower-casing the name seems reasonable enough if one isn't provided seems good.)

Reply:
Fixed. Please review.